.circleci/config.yml

```
@@ -16,6 +16,14 @@ jobs:
     - checkout
     - run: make promu
     - run: make check_license style unused staticcheck build check_assets
+    - run:
+       command: |
+         curl -s -L https://github.com/protocolbuffers/protobuf/releases/download/v3.5.1/protoc-3.5.1-linux-x86_64.zip > /tmp/protoc.zip
+         unzip -d /tmp /tmp/protoc.zip
+         chmod +x /tmp/bin/protoc
+         echo 'export PATH=/tmp/bin:$PATH' >> $BASH_ENV
+         source $BASH_ENV
+         make proto
     - run: git diff --exit-code
```

I assume this is to detected if assets, lint and now proto generation was done and commited, right? Maybe worth to wrap this with some message, like:
```suggestion
    - run: git diff --exit-code || echo "\n Not linted code, assets not updated or proto not generated.
```
It would be nice what exactly is not generated as well.. ;p But this will work for a start (: